packages/db-kms/src/db-kms.js

```diff
@@ -165,13 +169,43 @@ const initDbKms = (fastify, kmsOptions = {}) => {
       );
     };
 
+    /**
+     * encrypt text using a secret
+     * @param {Record<string, unknown>} plainText the text to encrypt
+     * @param {Id} keyId the key id to encrypt with
+     * @returns {string} the encrypted text
+     */
+    const encrypt = async (plainText, keyId) => {
+      const key = await loadKey(keyId);
+      const hex = hexFromJwk(
+        key[repoOptions.secretProp] ?? key[repoOptions.keyProp]
+      );
+      return _encrypt(plainText, hex);
+    };
+
+    /**
+     * Decrypt text using a secret
+     * @param {string} encrypted the encrypted text to decrypt
+     * @param {Id} keyId the key id to decrypt with
+     * @returns {string} the decrypted text
+     */
+    const decrypt = async (encrypted, keyId) => {
```

Comment:
as per the above

Reply:
done, [dac4604](https://github.com/velocitycareerlabs/velocitycore/pull/307/commits/dac46049a4ac96fef3ab4238eca6ae7d71279f71)